frontend/model/state.js

```diff
@@ -531,6 +531,11 @@ const getters = {
         const nameB = getters.ourContactProfilesByUsername[usernameB].displayName || usernameB
         return nameA.normalize().toUpperCase() > nameB.normalize().toUpperCase() ? 1 : -1
       })
+  },
+  isAppIdle (state) {
+    // NOTE: idleVue.isIdle is created by the IdleVue plugin
+    //       https://www.npmjs.com/package/idle-vue
+    return Boolean(state.idleVue?.isIdle)
```

Comment:
Based on the documentation from the plugin [here](https://github.com/soixantecircuits/idle-vue/tree/master?tab=readme-ov-file#example---mainjs-1), it indicates that `isAppIdle` is already accessible on all vue objects.

In other words, you don't need to create this getter, and you don't need to import it in `Background.vue`, you should be able to directly access `this.isAppIdle`.

Also, as it appears to be a boolean variable already, there's no need to wrap it in `Boolean`.

Reply:
Yeah, right. `this.isAppIdle` works fine as they described.
I thought it won't work because I couldn't find `isAppIdle` in the vuex state(`sbp('state/vuex/state')`).

Nice catch!